htdocs/product/inventory/inventory.php

```diff
@@ -1086,7 +1086,7 @@ function barcodeserialforproduct(tabproduct,index,element,barcodeproductqty,sele
 					print '<td class="right">';
 
 
-					if (! empty($obj->pmp_real)) $pmp_real = $obj->pmp_real;
+					if (! empty($obj->pmp_real) || $obj->pmp_real == 0) $pmp_real = $obj->pmp_real;
```

Comment:
If ( $obj->pmp_real) is null  (never set), $obj->pmp_real == 0 will return true when we want false.
May be you want to do
`if (! empty($obj->pmp_real) || (string) $obj->pmp_real === '0')`